tests/functional/aws-node-sdk/test/multipleBackend/put.js

```diff
@@ -113,6 +113,22 @@ describe('MultipleBackend put object', () => {
                     });
                 });
             });
+
```

Comment:
we should have tests where we are overwriting objects that were already put.  both 0 byte and non-0 byte.  also, overwriting to different backends.